src/main/java/seedu/address/storage/StorageMeeting.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.storage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.model.person.ReadOnlyAddressBook;
+import seedu.address.model.person.ReadOnlyUserPrefs;
+import seedu.address.model.person.UserPrefs;
+import seedu.address.storage.person.AddressBookStorage;
+import seedu.address.storage.person.UserPrefsStorage;
+
+/**
+ * API of the Storage component
+ */
+// TODO: build and change to MeetingBookStorage
+public interface StorageMeeting extends AddressBookStorage, UserPrefsStorage {
```

Comment:
Might need to take out UserPrefsStorage from Person Package.
AddressBookStorage seems to specific to the Person Model, what do you think?